src/ceph-volume/ceph_volume/devices/lvm/prepare.py

```diff
@@ -186,24 +200,23 @@ def prepare_device(self, arg, device_type, cluster_fsid, osd_fsid):
 
         :param arg: The value of ``--data`` when parsing args
         :param device_type: Usually, either ``data`` or ``block`` (filestore vs. bluestore)
-        :param cluster_fsid: The cluster fsid/uuid
-        :param osd_fsid: The OSD fsid/uuid
+        :param osd_uuid: The OSD uuid
         """
-        if disk.is_partition(arg) or disk.is_device(arg):
+        if disk.is_partition(device) or disk.is_device(device):
             # we must create a vg, and then a single lv
-            vg = api.create_vg(arg)
-            lv_name = "osd-%s-%s" % (device_type, osd_fsid)
+            lv_type = "osd-{}".format(device_type)
             return api.create_lv(
-                lv_name,
-                vg.name,  # the volume group
+                lv_type,
+                osd_uuid,
+                device=device,
                 tags={'ceph.type': device_type})
         else:
             error = [
-                'Cannot use device (%s).' % arg,
+                'Cannot use device ({}).'.format(device),
                 'A vg/lv path or an existing device is needed']
             raise RuntimeError(' '.join(error))
 
-        raise RuntimeError('no data logical volume found with: %s' % arg)
+        raise RuntimeError('no data logical volume found with: {}'.format(device))
```

Comment:
I don't mind what styling you prefer, but is this really necessary on such a large PR?

Reply:
I think there's an argument for it yes (readability) and I also can't think of an argument against doing this here.